.github/workflows/detect-breaking-changes-levitate.yml

```diff
@@ -235,7 +235,7 @@ jobs:
       # Check if label exists
       - name: Check if "levitate breaking change" label exists
         id: does-label-exist
-        uses: actions/github-script@v6
+        uses: actions/github-script@v7
```

Comment:
should probably be hash pinned?

Reply:
The [zizmor config in this repo](https://github.com/grafana/grafana/blob/main/.github/zizmor.yml#L2-L8) means none of the `actions/` are hash pinned in this repo. Seems a little odd to me, I'll ask around. 🤔